### PR TITLE
radio group component properly reflects ngModel updates

### PR DESCRIPTION
**Pull request by Blackbaud-AlexKingman**
Addresses #34

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -31,8 +31,8 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@blackbaud/skyux": "2.46.0",
+    "@blackbaud/skyux": "2.48.3",
     "@blackbaud/skyux-builder": "1.33.0",
```
Comment (Blackbaud-SteveBrush) [inline package.json]: Wanna bump this version of Builder to `1.34.1`?

File: src/app/public/modules/radio/radio-group.component.ts [hunk 2]
```diff
@@ -61,22 +61,16 @@ export class SkyRadioGroupComponent implements AfterContentInit, ControlValueAcc
...
+
+    if (isNewValue) {
       this._value = value;
+      this.onChange(this._value);
       this.updateCheckedRadioFromValue();
-
-      // Explicitly check both `undefined` and `null` (Angular's empty value) so that
-      // we can support boolean values for the form control.
-      /* tslint:disable:no-null-keyword */
-      if (
```
Comment (Blackbaud-TrevorBurch) [inline src/app/public/modules/radio/radio-group.component.ts]: I feel like these were here in order to make sure that the initial value doesn't set the `touched` or `dirty` flags. I might just be missing it but is there a test to ensure that we aren't accidentally setting those flags when an initial value is given? I see a test for when there isn't a value given but not for when there is.

File: package.json (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply package.json]: done!

File: src/app/public/modules/radio/radio-group.component.ts (hunk 2 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/radio/radio-group.component.ts]: You're right to look for that... Steve and I actually discussed that earlier. So, there's a known issue here with those states not being set correctly. It was already present in the code before this PR. We decided to work on that separately after.

As far as this particular code block goes, it was to ensure we were allowing boolean values. Steve made a PR to fix that a few weeks ago, and I confirmed that my changes still uphold the intent of the fix. While there was a test to look at the form control flags, it didn't necessarily cover the scenario where the initial value is set. I think that's a good idea, so I added it! Thanks for bringing that up.

Comment (Blackbaud-TrevorBurch) [reply src/app/public/modules/radio/radio-group.component.ts]: Thanks for adding that test!

**Pull request merged**